src/main/webapp/index.jsp

```diff
@@ -0,0 +1,82 @@
+<%@ page isELIgnored="false" language="java" contentType="text/html; charset=ISO-8859-1"
```

Comment:
Why not utf-8?
https://stackoverflow.com/questions/12723339/utf-8-encoding-in-jsp-page